.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,29 @@
-**Related issue**: <!-- if applicable -->
+**Related PRs / Issues **:
+<!-- List any related PRs/issues here, if applicable -->
 
 <!-- choose one -->
-**Type of change**: bug fix | new feature | other enhancement
+**Type of change**:
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Other enhancement
 
 <!-- choose one -->
-**Impact**: rtl change | software change | unknown | other
+**Impact**:
+- [ ] RTL change
+- [ ] Software change (RISC-V software)
+- [ ] Build system change
+- [ ] Other
+
+<!-- must be filled out completely to be considered for merging -->
+**Contributor Checklist**:
+- [ ] Yes, I specified the "Type of Change"
+- [ ] Yes, I specified the "Impact"
+- [ ] Did you set `dev` as the base branch?
```

Comment:
The first two bullet points here feel unnecessary, given that it will be quite obvious if the "Type of Change" and "Impact" sections are unfilled. 

Otherwise fine.

Reply:
I just tried making it extremely apparent that they/us need to fill out both sections... but honestly hasn't been an issue in the past. Done.